### PR TITLE
INSIGHTS-290 Call Team Bulk endpoint from cli

### PR DESCRIPTION
**Pull request by jdesouza**
This PR fixes #

## Checklist
* [x] I have signed the CLA
* [x] I have updated/added any relevant documentation

### CODE REVIEW

File: pkg/teams/teams.go [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+	}
+	if dryrun {
+		logrus.Infof("Dry run: Would have pushed the following teams configuration:")
+		for _, team := range teams {
+			logrus.Infof("Team: %s", team.Name)
+			logrus.Infof("  Clusters: %v", team.Clusters)
+			logrus.Infof("  DisallowedClusters: %v", team.DisallowedClusters)
+			logrus.Infof("  DisallowedNamespaces: %v", team.DisallowedNamespaces)
+			logrus.Infof("  DisallowedRepositories: %v", team.DisallowedRepositories)
+			logrus.Infof("  Namespaces: %v", team.Namespaces)
```
Comment (vitorvezani) [inline pkg/teams/teams.go:90]: [opt] push namespace and repositories up there with clusters, so it's separated allowed resources and disallowed resources

File: pkg/cli/push.go [hunk 2]
```diff
@@ -23,10 +23,12 @@ import (
 
 var pushDir string
 var pushDryRun bool
+var pushDeleteNonProvidedTeams bool
 
 func init() {
 	pushCmd.PersistentFlags().StringVarP(&pushDir, "push-directory", "d", ".", "Directory of content to push to Insights.")
 	pushCmd.PersistentFlags().BoolVarP(&pushDryRun, "dry-run", "z", false, "Explain what would be pushed to Insights, without making changes.")
```
Comment (vitorvezani) [inline pkg/cli/push.go:30]: Looks like this should be updated `Explain what would be pushed to Insights, without making changes.`

File: pkg/cli/push.go [hunk 3]
```diff
@@ -23,10 +23,12 @@ import (
...
+	pushCmd.PersistentFlags().BoolVarP(&pushDeleteNonProvidedTeams, "delete-non-provided-teams", "x", false, "Delete teams that are not provided in the push directory.")
```
Comment (vitorvezani) [inline pkg/cli/push.go:31]: Please, use `delete` instead to align with the other similar commands

Comment (jdesouza) [reply pkg/cli/push.go:31]: done

File: pkg/teams/teams.go (hunk 1 above)
Comment (jdesouza) [reply pkg/teams/teams.go]: done

**Pull request merged**